subsys/mgmt/mcumgr/bootutil_hooks/nrf53_hooks.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include "bootutil/bootutil_public.h"
+
+int boot_read_swap_state_primary_slot_hook(int image_index,
```

Comment:
now this is OK to have separate hook file for the mcumgr. Ther is already nrf/modules/mcuboot/hooks/nrf53_hooks.c file for mcuboot's hooks.
I think thanks to [PR](https://github.com/nrfconnect/sdk-nrf/pull/8133) this will be possible to be splited and reused in place of this copy.